src/Controllers/SubmitControllerCreate.php

```diff
@@ -88,4 +117,47 @@ function ($value)
 
 		return true;
 	}
+
+	/**
+	 * Check the CMS Version
+	 *
+	 * @return  false on failiure else the CMS Version
+	 *
+	 * @since   1.0
+	 */
+	private function checkCMSVersion($data)
+	{
+		$dotchecks = explode('.' $data);
+
+		// We ever use 2 dots and 3 parts in our CMS Version
+		if (count($dotchecks) != 3)
+		{
+			return false;
+		}
+
+		// The pulugin is installed since 3.5.0 other CMS Versions can't have this plugin installed
+		if (version_compare($data, '3.5.0', '<'))
```

Comment:
If we're doing that then we need to allow 5.3.3 and higher for PHP Version (cause of the minimum php version before 3.3)